bin/cheribsdtest/cheribsdtest.h

```diff
@@ -109,17 +108,10 @@ extern struct cheribsdtest_child_state *ccsp;
 					       but not checkable */
 #define CT_FLAG_SLOW		0x00000040  /* Test is expected to take a 
 					       long time to run */
-#define	CT_FLAG_SIGNAL_UNWIND	0x00000080  /* Should fault and unwind
-					       trusted stack; checks signum
-					       and result. */
-#define	CT_FLAG_SANDBOX		0x00000100  /* Test requires that a libcheri
-					     * sandbox be created. */
 #define	CT_FLAG_SI_CODE		0x00000200  /* Check signal si_code. */
```

Comment:
Should we move these values up to avoid having a gap?

Reply:
074fbca53c7016767b7216ead624730683661fb6 already left a hole